packages/wallet/ui/src/util/fixed-websocket-connector.js

```diff
@@ -17,71 +17,73 @@ export const waitForBootstrap = async getBootstrap => {
   return getBootstrap();
 };
 
-export const makeFixedWebSocketConnector = href => component => {
-  let ws;
+export const makeFixedWebSocketConnector =
+  ({ href }) =>
+  component => {
```

Comment:
since there's no type info, it helps to convey what the parameter is
```suggestion
export const makeFixedWebSocketConnector = config => component => {
  const { href } = config;
```
or make it `config.href`. or pass `href` to `makeFixedWebSocketConnector`!